src/animations/animatedImage/index.js

```diff
@@ -36,42 +44,67 @@ export default class AnimatedImage extends BaseComponent {
     /**
      * Use to identify the avatar in tests
      */
-    testId: PropTypes.string,
+    testId: PropTypes.string
   };
 
   static defaultProps = {
-    animationDuration: 300,
+    animationDuration: 300
   };
 
   constructor(props) {
     super(props);
     this.state = {opacity: new Animated.Value(0), isLoading: true};
+    this.checkForDeprecatedProps(props);
+  }
+
+  checkForDeprecatedProps(props) {
+    deprecatedProps.forEach((prop) => {
+      if (props[prop.old]) {
+        console.warn(`'${prop.old}' property is deprecated, use '${prop.new}' instead`);
+      }
+    });
+  }
+
+  get source() {
+    const {imageSource, source} = this.props;
+    return source || imageSource;
+  }
+
+  get style() {
+    const {imageStyle, style} = this.props;
+    return style || imageStyle;
+  }
+
+  get testID() {
+    const {testId, testID} = this.props;
+    return testID || testId;
   }
 
   onLoad = () => {
     this.setState({isLoading: false}, () => {
       const animationParams = {toValue: 1, duration: this.props.animationDuration, useNativeDriver: false};
       Animated.timing(this.state.opacity, animationParams).start();
     });
-  }
+  };
 
   render() {
-    const {containerStyle, imageStyle, imageSource, loader, testId} = this.props;
+    const {containerStyle, loader, ...others} = this.props;
     return (
-      <View style={containerStyle} testID={testId}>
-        <Animated.Image
-          style={[{opacity: this.state.opacity}, imageStyle]}
-          source={imageSource}
-          onLoad={this.onLoad}
+      <View testID={this.testID} style={containerStyle}>
+        <UIAnimatedImage
+          {...others}
+          style={[{opacity: this.state.opacity}, this.style]}
+          source={this.source}
+          onLoad={() => this.onLoad()}
         />
-        {this.state.isLoading && loader &&
-          <View style={{position: 'absolute', top: 0, left: 0, bottom: 0, right: 0, alignItems: 'center'}}>
-            <View style={{flex: 1, flexDirection: 'row', alignItems: 'center'}}>
-              {loader}
-            </View>
+        {this.state.isLoading && loader && (
+          <View style={{position: 'absolute', top: 0, left: 0, bottom: 0, right: 0, justifyContent: 'center'}}>
```

Comment:
use `StyleSheet.absoluteFillObject` to get this preset : `position: 'absolute', top: 0, left: 0, bottom: 0, right: 0`